src/devices/philips.ts

```diff
@@ -631,6 +631,13 @@ const definitions: DefinitionWithExtend[] = [
         description: 'Hue white A67 bulb E26 with Bluetooth (1600 Lumen)',
         extend: [philipsLight()],
     },
+    {
+        zigbeeModel: ['LWA030'],
+        model: 'LWA030',
```

Comment:
What's the model number on the product/site?

Reply:
on the box, it says 9290023351B

Reply:
updated